chain/jsonrpc/res/epoch_info.html

```diff
@@ -208,12 +256,39 @@ <h3><span class="js-next-epoch"></span></h3>
                 <th>Protocol version</th>
                 <th>First block</th>
                 <th>Epoch start</th>
+                <th>Block producers</th>
+                <th>Chunk only producers</th>
```

Comment:
oh - and we could also compute the number of block and chunks missed during the whole epoch, right?

Reply:
uh - unfortunately we cannot do that..

As EpochInfo (for some reason) keeps only the stats from the producers that were not kicked out:
        /// Only for validators who met the threshold and didn't get slashed
        pub validator_block_chunk_stats: HashMap<AccountId, BlockChunkValidatorStats>,

Reply:
We could, if we record this information somewhere. It's not too difficult to compute. I can do that in another PR.